POC/ColorPicker/ColorPicker/App.xaml

```diff
@@ -4,6 +4,6 @@
              xmlns:local="clr-namespace:ColorPicker"
              StartupUri="MainWindow.xaml">
     <Application.Resources>
-         
+
```

Comment:
Application resources tag is empty, should probably be removed

Reply:
This is the generated boilerplate so I left it alone, all I did was remove the trailing whitespace.